pkg/registry/scripts/images.js

```diff
@@ -152,11 +158,25 @@
                 return { };
             };
 
-            angular.extend($scope, data);
-
             $scope.$on("$destroy", function() {
                 c.cancel();
             });
+
+            /* All the data actions available on the $scope */
+            angular.extend($scope, data);
+            angular.extend($scope, actions);
+
+            /* But special case a few */
```

Comment:
Is this comment a continuation of the previous one? The phrasing seems a bit opaque.